spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -235,11 +225,11 @@
       expect{
         api_post :find_or_create,
                  trusted_application_token,
-                 raw_post_data: {email: 'a-new-email@test.com'}
+                 raw_post_data: {email: 'a-new-email@test.com', first_name: 'Ezekiel', last_name: 'Jones'}
       }.to change{User.count}.by(1)
       expect(response.code).to eq('201')
       new_user_id = User.order(:id).last.id
-      expect(response.body).to eq({id: new_user_id}.to_json)
+      expect(response.body).to eq({'id'=> new_user_id}.to_json)
```

Comment:
ah, yes it is. I had an earlier failure on that line where i was using the new `users_hash` method here.  Once I figured out that wasn't needed, it looks like I updated the line to use string keys.  I've set it back now.